lib/core/src/server/config/utils.js

```diff
@@ -25,7 +25,16 @@ const projectRoot = () => {
 
 export const includePaths = [projectRoot()];
 export const nodeModulesPaths = path.resolve('./node_modules');
-export const excludePaths = [/node_modules/];
+
+// Run `npx are-you-es5 check lib/core -r`
+// to get this regexp
+export const excludePaths = [
+  /[\\/]node_modules[\\/](?!(@storybook\/node-logger|are-you-es5|better-opn|boxen|chalk|commander|find-cache-dir|find-up|fs-extra|json5|node-fetch|pkg-dir|resolve-from|semver)[\\/])/,
```

Comment:
why is `@storybook/node-logger` in this list?

Reply:
I unfortunately don't know. I was surprised as well but also didn't wanted to alter the output of the regexp

Reply:
Because it only is run in node, and so ES6 is fine?

Reply:
it should not be bundled into client-code ever

Reply:
being in this regexp doesn't mean you'll get bundled. It's only "If you're being bundled and going though babel, you'll get transpiled"

Reply:
of course, the `are-you-es5` just detects the code being ES6 and adds it to the list.